.github/workflows/main.yml

```diff
@@ -0,0 +1,38 @@
+name: Main

```

Comment:
Our other libraries use build, but I've been noticing a pattern of using `main` for the only or primary workflow.